docs(other): add screenshots for navigation elements (#310)
Redocly/website: docs/end-user/interact-with-pages.md

  You can use this initial prompt to provide context for the conversation.
  This action is available on publicly accessible pages only.
  Requires a ChatGPT account.
- **Claude** or **Open in Claude**: - opens the Claude website in a new tab with the page's URL in the prompt.
  You can use this initial prompt to provide context for the conversation.
  This action is available on publicly accessible pages only.
  Requires a Claude account.
- **Connect to Cursor** - opens the Cursor application and installs the MCP server in your local environment.

> - **Connect to VS Code** - opens the Cursor application and installs the MCP server in your local environment.

Author: ```suggestion
- **Connect to VS Code** - opens the Visual Studio Code application and installs the MCP server in your local environment.
```

To copy a code sample:

> - In the top-right corner of the code sample, click the copy  icon.

Author: ```suggestion
- In the top-right corner of the code sample, click the copy icon.
```

Your message is sent to the organization that owns the project.

    {% img
      src="./images/report-code-snippet.png"

>       alt="Screenshot of a code block with exclamation mark icon marked""

Author: ```suggestion
      alt="Screenshot of a code block with exclamation mark icon marked"
```
